### ISSUE TITLE
Allow renaming image directory and image.zip file

### ISSUE THREAD

**Issue by jperedadnr** [beryx/badass-jlink-plugin#17]
So far, the image dir is named `image`, and the produced zip is named `image.zip`.

While that can be changed:

```
jlink {
    imageDir.set(project.layout.buildDirectory.dir('myImage'))
    imageZip.set(project.layout.buildDirectory.file('myImage.zip'))
...
}
```

it should be more convenient using a property that could be set like:
```
jlink {
    imageDirName='myImage'
    imageZipName='myImage.zip'
...
}
```

**Comment by siordache** [comment]
Try something like this:

```
jlink {
    imageDir = file("$buildDir/myImage")
    imageZip = file("$buildDir/myImage.zip")
}
```

See an example [here](https://github.com/beryx-gist/badass-jlink-spring-petclinic/blob/d91571341e7d5c241d1b91ecd1ef4ebf2eb43eff/build.gradle#L99-L100).

**Comment by jperedadnr** [comment]
Yes, right, I saw that before, but I didn't recall about it, thanks for pointing it out.

Anyway, having maybe just a String property with the name of the image would be more convenient and more clean for the user:
```
jlink {
    imageName = 'myImage'
}
```
that will generate the folder `myImage`, and the zip `myImage.zip`.

**Comment by siordache** [comment]
Yes, that's a good suggestion.

**Comment by siordache** [comment]
Implemented in 2.3.0
